### PR TITLE
Set up CORS allowed origins

### PR DESCRIPTION
**Pull request by brittanylau**
### Notion ticket link

[Set up API helper functions (frontend)](https://www.notion.so/uwblueprintexecs/9cb454262ed84c6299c0682c23f92ec5?v=1076bd30711a4a01838becee05f8b5a3&p=e549fca73569467fa31a0270e91815d1)

### Implementation description

- install CORS middleware
- if in debug mode, allow traffic from localhost:3000

### Steps to test

1. test together with [the frontend PR](https://github.com/uwblueprint/project-read-frontend/pull/4)

### What should reviewers focus on?

- do we need to allow more hosts?

### Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] I have run the linter
- [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by jolenezheng — APPROVED** [review]
I don't think we need to allow for more hosts? But maybe I'm just missing a use case where we would/should add more.

LGTM! 👉🏼🐸👉🏼

**Comment by Tayef-Shah** [comment]
@jolenezheng which line are you talking about?

File: projectread/settings.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
 ALLOWED_HOSTS = [] if DEBUG else ["project-read-backend.herokuapp.com"]
 
+CORS_ALLOWED_ORIGINS = ["http://localhost:3000"] if DEBUG else []
```
Comment (Tayef-Shah) [inline projectread/settings.py:35]: would the else clause just have the origin for our front-end in the future? (I'm new to CORS btw)

**Review by Tayef-Shah — APPROVED** [review]

File: projectread/settings.py (hunk 1 above)
Comment (brittanylau) [reply projectread/settings.py:35]: yep that's right! we'll have to add the url when we deploy it

**Pull request merged**